Content.Server/GameObjects/Components/Chemistry/ChemMasterComponent.cs

```diff
@@ -257,14 +256,14 @@ private void TransferReagent(string id, ReagentUnit amount, bool isBuffer)
                         ReagentUnit actualAmount;
                         if (amount == ReagentUnit.New(-1))
```

Comment:
This -1 check threw me for a loop, but now I see it's because -1 is passed when clicking the "All" button for the quantity, which is unintuitive behavior. Please add a comment clarifying this (and above)